### DISCUSSION TITLE
Acces point issues with electrodragon LED control board (ESP32-C3-WROOM-2)

### DISCUSSION THREAD

**Discussion by DerDom** [arendst/Tasmota#18491] [Q&A] [answered]
Hey,

I am able to flash tasmota on my electrodragon LED control board (ESP32-C3-WROOM-2) using https://tasmota.github.io/install via the usb socket on the control board withouth any issues. However, after the installation the access point does not show up reliably (unfortunately, I cannot tell when it appears and when it doesn't). If it happens to show up and if I am able to connect to the tasmota_XXXX wifi I cannot access the setup page http://192.168.4.1. I managed to do it one single time but most of the time my Chrome browser tells me the website is not accessible.

I already tried different versions of tasmota onto the boards. I belive I got to the setup page with the "Tasmota for 4MB flash (english)".

Does anybody know what might help me to finish the tasmota setup? It so frustratig since I made it almost through all the steps but cannot replicate it anymore.

**Comment by DerDom** [comment, marked as answer]
development Tasmota32 bluetooth (english) works

**Comment by Jason2866** [comment]
The Tasmota32 C3 build does work.

**Comment by Kepete** [comment]
Similar problem. Access point is available, but it fails when joining it. 
Tasmota32 C3 tried using https://tasmota.github.io/install/ and esp flasher. 

Anyone had a similar problem and found a solution?

**Reply by Jason2866** [reply]
@Kepete Either bad power supply or an esp32c3 with bad antenna.

**Reply by sfromis** [reply]
It is very common that the smallest ESP32-xx boards have a bad antenna design.

**Reply by Kepete** [reply]
Interesting, indeed seemed to be this problem. Once I changed to a second board and flashed with Tasmota-C3 it worked. Have flashed tens and tens of sonoff, H801, magichome and nodemcu boards without any antenna problems. 

Thanks for the hint!
